documentation/api/query/v4/nodes.markdown

```diff
@@ -59,6 +68,24 @@ The below fields are allowed as filter criteria and are returned in all response
     Note that nodes which are missing a fact referenced by a `not` query will match
     the query.
 
+### Subquery Relationships
+
+Here is a list of relationships that can be queried using implicit subqueries. For
+more information consult the documentation for [subqueries].
+
+Direct relationships:
+
+* [factsets][`factsets`]: Factsets received from this node.
+* [reports][`reports`]: Reports received from this node.
+* [catalogs][`catalogs`]: Catalogs received for this node.
+
+Transitive relationships:
+
+* [facts][`facts`]: Fact names and values received from this node.
+* [fact-contents][`fact_contents`]: Factset paths and values received from this node.
+* [events][`events`]: Report events triggered from this node.
+* [edges][`edges`]: Catalog edges received for this node.
+* [resources][`resources`]: Catalog resources received for this node.
```

Comment:
this bracketed style renders badly in github markdown -- not sure how it'll look on the web, but it's been introduced in a number of places here.